tests/foreman/api/test_template.py

```diff
@@ -1405,7 +1405,7 @@ def test_positive_import_log_to_production(self):
         self.assertIn(
             'Started POST "/api/v2/templates/import"',
             ssh.command(
-                'tail -10 /var/log/foreman/production.log | grep import'
+                'tail -100 /var/log/foreman/production.log | grep import'
```

Comment:
maybe the bulletproof way would be to grep from everything after a timestamp of when the template started, but if you're sure enough 100 is safe enough buffer, than ack

Reply:
Yeah, That would be the great way as well, I will test that and update the PR if it works.